perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -204,54 +204,38 @@ def _get_start_end_times(self, out_json, job_params) -> List[Tuple[int]]:
       KeyError: If RW is not present in any dict in job_params
 
     """
-    # Creating a list of just the 'rw' job parameter. Later, we will
-    # loop through the jobs from the end, therefore we are creating
-    # reversed rw list for easy access
-    rw_rev_list = [job_param[consts.RW] for job_param in reversed(job_params)]
+    # Creating a list of just the 'rw' job parameter.
+    rw_list = [job_param[consts.RW] for job_param in job_params]
 
     global_ramptime_ms = 0
-    global_startdelay_ms = 0
     if consts.GLOBAL_OPTS in out_json:
       if consts.RAMPTIME in out_json[consts.GLOBAL_OPTS]:
         global_ramptime_ms = _convert_value(
             out_json[consts.GLOBAL_OPTS][consts.RAMPTIME],
             consts.TIME_TO_MS_CONVERSION, 's')
-      if consts.STARTDELAY in out_json[consts.GLOBAL_OPTS]:
-        global_startdelay_ms = _convert_value(
-            out_json[consts.GLOBAL_OPTS][consts.STARTDELAY],
-            consts.TIME_TO_MS_CONVERSION, 's')
 
-    next_end_time_ms = 0
-    rev_start_end_times = []
-    # Looping from end since the given time is the final end time
-    for i, job in enumerate(list(reversed(out_json[consts.JOBS]))):
-      rw = rw_rev_list[i]
+    start_end_times = []
+    for i, job in enumerate(list(out_json[consts.JOBS])):
+      rw = rw_list[i]
       job_rw = job[_get_rw(rw)]
       ramptime_ms = 0
-      startdelay_ms = 0
       if consts.JOB_OPTS in job:
         if consts.RAMPTIME in job[consts.JOB_OPTS]:
           ramptime_ms = _convert_value(job[consts.JOB_OPTS][consts.RAMPTIME],
                                        consts.TIME_TO_MS_CONVERSION, 's')
 
       if ramptime_ms == 0:
         ramptime_ms = global_ramptime_ms
-      if startdelay_ms == 0:
-        startdelay_ms = global_startdelay_ms
 
-      # for multiple jobs, end time of one job = start time of next job
-      end_time_ms = next_end_time_ms if next_end_time_ms > 0 else out_json[
-          consts.TIMESTAMP_MS]
-      # job start time = job end time - job runtime - ramp time
-      start_time_ms = end_time_ms - job_rw[consts.RUNTIME] - ramptime_ms
-      next_end_time_ms = start_time_ms - startdelay_ms
+      start_time_ms = job[consts.JOB_START]
```

Comment:
Should we throw error in case "start_time" does not exist in the output_json?

Reply:
if job_start won't be present in output, it will raise key error.

Eg:
```
Traceback (most recent call last):
  File "/usr/local/google/home/ashmeen/gcsfuse/perfmetrics/scripts/fio/fio_metrics_test.py", line 430, in test_extract_metrics_from_good_file
    extracted_metrics = self.fio_metrics_obj._extract_metrics(json_obj)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/google/home/ashmeen/gcsfuse/perfmetrics/scripts/fio/fio_metrics.py", line 356, in _extract_metrics
    start_end_times = self._get_start_end_times(fio_out, job_params)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/google/home/ashmeen/gcsfuse/perfmetrics/scripts/fio/fio_metrics.py", line 237, in _get_start_end_times
    start_time_ms = job[consts.JOB_START]
                    ~~~^^^^^^^^^^^^^^^^^^
KeyError: 'job_start'

 ```